src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileTitleBarViewImpl.java

```diff
@@ -224,7 +218,12 @@ public void setVersion(Long versionNumber) {
 	@Override
 	public void setCanDownload(boolean canDownload) {
 		downloadOptionsButton.setEnabled(canDownload);
-		viewOnlyHelp.setVisible(!canDownload);
+		if(!canDownload){
+			String viewOnlyHelpText = authController.isLoggedIn() ? "You do not have download access for this item." : "You need to log in to download this file.";
+			downloadTooltip.setTitle(viewOnlyHelpText);
+		} else{
+			downloadTooltip.setTitle("Direct and programmatic download options");
+		}
```

Comment:
Just recommend flipping the cases for readability
```suggestion
		if (canDownload) {
			downloadTooltip.setTitle("Direct and programmatic download options");
		} else{
			String viewOnlyHelpText = authController.isLoggedIn() ? "You do not have download access for this item." : "You need to log in to download this file.";
			downloadTooltip.setTitle(viewOnlyHelpText);
		}
```